.idea/workspace.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Was adding this file intentional? Ignore me if so :) Just happened to notice...

Reply:
Uh, good one.
I didn't add the .idea to the `.git/info/exclude`

Gonna remove them, thanks for noticing!